.github/workflows/clean-workflow-logs.yml

```
@@ -0,0 +1,27 @@
+  SCHEDULED_RUNS_TO_KEEP: "0"
+
+jobs:
+  clean-logs:
+    runs-on: ubuntu-latest
+    permissions:
+      actions: write
+    steps:
+      - uses: igorjs/gh-actions-clean-workflow@v6
+        with:
+          runs_older_than: ${{ github.event.inputs.runs_older_than || env.SCHEDULED_RUNS_OLDER_THAN }}
+          runs_to_keep: ${{ github.event.inputs.runs_to_keep || env.SCHEDULED_RUNS_TO_KEEP }}
```

actionlint (1.7.4)

property "runs_to_keep" is not defined in object type {runs_older_than: string}

(expression)